### DISCUSSION TITLE
Gain a better understanding of why users are users!

### DISCUSSION THREAD

**Discussion by Felienne** [hedyorg/hedy#3915] [Community]
Currently, we don't really know what drives people to use start to use Hedy (or to keep using, or stop) and we should get some more insights into that. Maybe by making a short survey at registration, and one to send after 1 and then 6 months or something?

Or (a bit more advanced and requiring some programming) send users dedicated emails with one or two questions in interesting situations, like when they have not visited in a while.

**Comment by woutccc** [comment]
@Felienne is it correct that main focus for 2023 is "the teacher" and make adoption of Hedy in the classroom as easy and fun as possible?

**Reply by Felienne** [reply]
Yes that is a great summary!

**Comment by Mark-Giesen** [comment]
Maybe we could integrate it by asking the user to fill in some goals at the start. We could provide 5 predefined and one "other" option. Just like hunting for batches, we could try to nudge them to reach those goals.  If they're absent for a while, we could nudge them via mail, but only then and maybe a little personalized by asking them about the missing goals.

**Comment by woutccc** [comment]
When optimizing Hedy for teachers is primary goal, I think we should start reaching out to teachers that stopped using Hedy, and focus on 3 groups:
1. Signed up but never really started (what blocked you from starting?)
2. Signed up, started but stopped in the first 6 months (what blocked you from following through?)
3. Signed up, followed through for a full school year and then stopped (what made you decide not to continue for another year)
When reaching out to them, we could ask them what their initial goal was when starting Hedy.

**Reply by Mark-Giesen** [reply]
That would be great information, maybe even a fourth group: Signed up and still using, what would they need to start a new class next year or make teaching easier. This group might even have interesting content that would be worth to share and/or integrate in Hedy.

**Reply by Felienne** [reply]
> When optimizing Hedy for teachers is primary goal, I think we should start reaching out to teachers that stopped using Hedy, and focus on 3 groups:
> 
> 1. Signed up but never really started (what blocked you from starting?)
> 2. Signed up, started but stopped in the first 6 months (what blocked you from following through?)
> 3. Signed up, followed through for a full school year and then stopped (what made you decide not to continue for another year)
>    When reaching out to them, we could ask them what their initial goal was when starting Hedy.

That sounds good! I will discuss the implementation of this on the code side tomorrow in the meetup! Maybe @jpelay has time to pick this up. or I can do it myself too.

**Reply by woutccc** [reply]
yes I can create a set of questions (as short and simple as possible :-)) per survey type.

**Comment by woutccc** [comment]
Hi, I made a first draft, is not what it needs to be yet but just checking with you if this is the right approach. It's a draft of the first survey type (teacher signed up, but never used Hedy after that). Please leave comment in-doc; https://docs.google.com/document/d/1kEQWoiExg9tWbqe7Cf3vUGIrWHFHENpP1yIrgyZkmJQ/edit?usp=sharing

**Reply by Felienne** [reply]
Thanks a lot @woutccc!! I made a few notes but I think it looks great!
